### PR TITLE
Update nav bar

### PR DESCRIPTION
**Pull request by cellomatt**
Please merge the nav bar!

### CODE REVIEW

File: aurora-project-starter/views/question-ask.pug [hunk 1]
```diff
@@ -4,25 +4,27 @@ append head
...
-        input(type="hidden" name="_csrf" value=csrfToken)
-        label(for="title") Your Question:
-        div
-        input(type="text" name="title" class="form__input_field" required)
+    div.parent
+        form(action="/questions" method="post" class="form child")
+            h1.form__title Add a Question
+            input(type="hidden" name="_csrf" value=csrfToken)
+            label(for="title") Your Question:
+            div
```
Comment (genagain) [inline aurora-project-starter/views/question-ask.pug:12]: Can you please tell me why you have this `div` here?

File: aurora-project-starter/views/question-ask.pug [hunk 2]
```diff
@@ -4,25 +4,27 @@ append head
...
+                    option(value=expertise.id class="form__dropdown--option")=expertise.description
+            div
 
-        label(for="message") Additional Information:
-        div
-        textarea(name="message" class="form__input_field textarea" required)
-        div
-        button(type="submit" class="form__submit_button") Submit
+            label(for="message") Additional Information:
+            div
```
Comment (genagain) [inline aurora-project-starter/views/question-ask.pug:27]: the above applies here too

File: aurora-project-starter/views/question-ask.pug [hunk 3]
```diff
@@ -4,25 +4,27 @@ append head
...
+            textarea(name="message" class="form__input_field" placeholder="Add context to your question here..." required)
+            div
```
Comment (genagain) [inline aurora-project-starter/views/question-ask.pug:29]: the above applies here as well

File: aurora-project-starter/views/login.pug [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
-      input(type="hidden" name="_csrf" value=csrfToken)
-      div.small_hyperlink
-        a(href='/signup') Don't have an account? Sign up here.
+  div.parent
+    div.form.child
+      h1.form__title Log In
+      form(action='/login' method='post')
+        label(for='username') Username:
+        input(type="text" name="username" class="form__input_field" required)
+        div
```
Comment (genagain) [inline aurora-project-starter/views/login.pug:13]: Here as well

File: aurora-project-starter/views/login.pug [hunk 5]
```diff
@@ -1,17 +1,20 @@
...
+        label(for="password") Password:
+        input(type="password" name="password" class="form__input_field" required)
+        div
```
Comment (genagain) [inline aurora-project-starter/views/login.pug:16]: Here too

File: aurora-project-starter/views/signup.pug [hunk 6]
```diff
@@ -1,25 +1,28 @@
...
-      div.small_hyperlink
-        a(href='/login') Already have an account? Log in here.
-      input(type="hidden" name="_csrf" value=csrfToken)
+  div.parent
+    div.form.child
+      h1.form__title Sign Up
+      form(action="/signup" method="post")
+        label(for="username") Username:
+        input(type="text" name="username" class="form__input_field" placeholder="Your username.." required)
+        div
```
Comment (genagain) [inline aurora-project-starter/views/signup.pug:14]: Here too.

File: aurora-project-starter/views/signup.pug [hunk 7]
```diff
@@ -1,25 +1,28 @@
...
+        label(for="email") Email:
+        input(type="email" name="email" class="form__input_field"  placeholder="Your email.." required)
+        div
```
Comment (genagain) [inline aurora-project-starter/views/signup.pug:17]: Here too.

File: aurora-project-starter/views/signup.pug [hunk 8]
```diff
@@ -1,25 +1,28 @@
...
+        label(for="password") Password:
+        p.form__sub_label Please include at least one capital letter, one lowercase letter, one number, and one special character (!@#$%^&*).
+        input(type="password" name="password" class="form__input_field" placeholder="Your super secret password!" required)
+        div
```
Comment (genagain) [inline aurora-project-starter/views/signup.pug:21]: Here too

File: aurora-project-starter/views/signup.pug [hunk 9]
```diff
@@ -1,25 +1,28 @@
...
+        label(for="confirmPassword") Confirm Password:
+        input(type="password" name="confirmPassword" class="form__input_field" placeholder="One more time.." required)
+        div
```
Comment (genagain) [inline aurora-project-starter/views/signup.pug:24]: Here too

**Review by genagain — COMMENTED** [review]
It looks like there are some extra `div`s near the input fields and I'm not sure that they are necessary. Can you please tell me why they are there

**Review by lettemax — APPROVED** [review]
k!

**Pull request merged**